log/wrapper.go

```diff
@@ -273,7 +272,15 @@ func ErrorWithSentryWrapper() Wrapper {
 	}
 }
 
-// PrometheusCounterWrapper returns a Wrapper implementation that increases
+// Counter is a minimal interface for a counter.
+//
+// This is implemented by both prometheus counter and statsd counter from
+// metricsbp.
+type Counter interface {
```

Comment:
Does this need to be exported though?

Reply:
it's required by exported function `CounterWrapper` so yes it needs to be exported.